lib/source/mqtt/iot_mqtt_network.c

```diff
@@ -727,6 +751,23 @@ static void _sendPuback( _mqttConnection_t * pMqttConnection,
 
 /*-----------------------------------------------------------*/
 
+static void _flushPacket( void * pNetworkConnection,
+                          const _mqttConnection_t * pMqttConnection,
+                          size_t length )
+{
+    size_t bytesFlushed = 0;
+    uint8_t receivedByte = 0;
+
+    for( bytesFlushed = 0; bytesFlushed < length; bytesFlushed++ )
+    {
+        ( void ) _IotMqtt_GetNextByte( pNetworkConnection,
```

Comment:
Is the returning status important here?

Reply:
No, can't do anything because there's no memory to process the packet.  It's discarded so the next packet may be processed.